activerecord/test/cases/attribute_methods_test.rb

```diff
@@ -163,13 +163,13 @@ def setup
     assert_equal "10", keyboard.read_attribute_before_type_cast(:key_number)
   end
 
-  # Syck calls respond_to? before actually calling initialize.
   test "respond_to? with an allocated object" do
```

Comment:
I think "allocated object" here specifically means "allocated and not yet initialized", per the comment about Syck. If that's no longer something we need to care about, seems like the whole test can just go away?

Reply:
Ya, agree.  I was debating whether or not to just rm the whole test.  I will rm the whole test now. 😉